test/specs/plugin.tooltip.tests.js

```diff
@@ -20,7 +20,7 @@ describe('Plugin.Tooltip', function() {
 				datasetIndex: 0,
 				dataset: data.datasets[0],
 				label: 'Point 2',
-				value: '20'
+				formattedValue: '20'
 			};
```

Comment:
Might want to add the dataPoint in this test too?

Reply:
done

Reply:
Thought it would be an object, perhaps the test was not a good one for that. Anyway, I was thinking it should be in a test to prevent regression later.
So maybe if you can find/ctreate another test where it would be an object, it would be good to add.

Reply:
Ok, I added a new test where it's an object